src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultArray = {};
+  const splitString = sourceString.split(';');
+
+  splitString.map((element) => {
+    if (element.length >= 2) {
+      const splitElement = element.split(':').map((item) => item.trim());
+
+      resultArray[splitElement[0]] = splitElement[1];
+    }
+  });
```

Comment:
The 'map' method is used incorrectly here. 'map' should be used when you want to transform an array into another array of the same length. Since you're not using the returned array from 'map', a 'forEach' method would be more appropriate in this case.